cmd/vinegar/vinegar.go

```diff
@@ -91,6 +92,12 @@ func main() {
 			if err := InstallWebview2(&pfx); err != nil {
 				log.Fatal(err)
 			}
+		case "reportinfo":
+			report, err := extra.GenerateInfo(cfg)
```

Comment:
Pass configuration as pointer to prevent more memory usage or garbage collection

Reply:
It's already a pointer

Reply:
i lied